libs/wallet-ui/src/components/connection-list/connection-manage.tsx

```diff
@@ -81,16 +87,16 @@ const compileSubmissionData = (
 ): WalletModel.Permissions => {
   const dataKeys = Object.keys(formData) as Array<keyof WalletModel.Permissions>
   return dataKeys.reduce<WalletModel.Permissions>((acc, key) => {
-    const p = formData[key]
+    const p = formData[key] as NormalizedPermission
     return {
       ...acc,
       [key]: {
-        access: p.access,
+        access: p.access === 'read-all' ? 'read' : p.access,
```

Comment:
nitpick: magic strings should probably be in an enum or object

Reply:
so if rename is required we don't have to find it everywhere

Reply:
the reason i've been using the string versions was because the generator compiles the access type to 
```
access: 'read' | 'none'
```

and that's like a painful extra task to transform back and forth when trying to use the generated permission types, so i'll leave this as it is for now